feature-utils/silly-i18n/rollup-plugin.js

```diff
@@ -0,0 +1,57 @@
+import fs from "fs";
+import path from "path";
+import util from "util";
+import glob from "glob";
+
+const DIRNAME = "locales";
+const FILEEXT = ".json";
+const PLUGIN_NAME = "silly-i18n";
+const SUFFIX = `?${PLUGIN_NAME}`;
+
+export default function (options) {
+    return {
+        name: PLUGIN_NAME,
+
+        async resolveId(source, importer) {
+            if (source == `!${PLUGIN_NAME}`) {
+                for (let dir = importer; dir != (dir = path.dirname(dir)); ) {
+                    const subdir = path.join(dir, DIRNAME);
+                    const exists = await fs.promises.stat(subdir).then(
+                        (stat) => stat.isDirectory(),
```

Comment:
Where are we using this `stat`?

Reply:
Right there, in this very line. What do you mean?

Reply:
I mean where are we using this function with the stat argument.

Reply:
It's an anonymous function passed to the `then`  method of a `Promise` right here, i.e. it's called as this promise is resolved successfully, creating another promise with the result of that function.